src/styles/palette.js

```diff
@@ -4,7 +4,7 @@ const variant = {
   primary: color.blue,
   info: color.blueDark,
   success: color.green,
-  warning: color.yellow,
+  warning: color.black,
```

Comment:
I think that this one should be kept as yellow but in the `Alert` component the text should be changed to black instead of yellow, which it is today. This can be done like this: 
```

Reply:
Ah, I didn't notice buttons broke. I checked them on my local, but likely before I replaced the `palette.warning`.

Should be fixed with my latest push:
![black text warning alert](https://user-images.githubusercontent.com/235857/35734899-0036364c-0823-11e8-877c-4b9058f256b8.png)

![warning buttons](https://user-images.githubusercontent.com/235857/35734898-fe5e11dc-0822-11e8-80a1-3c7761d25689.png)